samples/cellular/nrf_cloud_multi_service/src/cloud_connection.c

```diff
@@ -270,11 +299,11 @@ static void l4_event_handler(struct net_mgmt_event_callback *cb,
 	if (event == NET_EVENT_L4_CONNECTED) {
 		LOG_INF("Network connectivity gained!");
 
-		k_sleep(K_MSEC(1000));
-
 		/* Set the network ready flag */
 		k_event_post(&cloud_events, NETWORK_READY);
 
+		k_sleep(K_MSEC(1000));
+
```

Comment:
What was the reason for moving this?

Reply:
Might not be necessary. I did that while trying to improve handling of "bouncing" network connections.